src/ca350

```diff
@@ -75,11 +75,24 @@ def debug_data(serial_data):
 def on_message(client, userdata, message):
     #print("message check")
     msg_data = str(message.payload.decode("utf-8"))
-    json_data = json.loads(msg_data)
-    print(message.topic)
-    print(msg_data)
     fan_level = -1
-    if message.topic == "comfoair/speed/set":
```

Comment:
I guess that is a breaking change for my way of using it, I'll need to adjust that on my HA config - ideally use all new auto discovered entities in my setup and automations. Correct?

Reply:
actually it should't, the "comoair/speed/set" is still there but moved down to line 95. Otherwise I will look into it!

Reply:
Ah, right, sorry I didn't check it there :)